kalite/distributed/static/js/distributed/search/views.js

```diff
@@ -116,6 +116,14 @@ window.AutoCompleteView = BaseView.extend({
             return compvalue;
         });
 
+        // Then sort again by availability!
+        ids_filtered.sort(function(id1, id2) {
```

Comment:
Right, the stability of Array.prototype.sort depends on the browser. Will use a stable sort.